v3/setup/capcom.sh

```diff
@@ -7,26 +7,24 @@ if [ "${NODE_ROLE}" != "control" ]  && [ "${NODE_ROLE}" != "it-hybrid" ]; then
     exit 0
 fi
 
-if [ "$(etcdctl get capcom-bootstrapped)" == "true" ]; then
+if [ "$(etcdctl get /capcom/config/bootstrapped)" == "true" ]; then
```

Comment:
hm good point. I was reading it more as `has the capcom config been bootstrapped?`. Hence the order & namespace.

Do you think it should look like something `/cluster-state/capcom/bootstrapped`?